### ISSUE TITLE
does this suitable for rt2880f in dlink 685? does it need ap/sta firmare?

### ISSUE THREAD

**Issue by youxiaojie** [Quallenauge/ralink_inic#3]
does this suitable for rt2880f in dlink 685? does it need ap/sta firmare?

**Comment by Quallenauge** [comment]
No, this is not an generic approach to support multiple devices but it is really specific support driver for the rt3883 chipset, embedded in the EasyBox 904 xDSL device. Also the firmware is device specific.
